Protocol.md

```diff
@@ -377,18 +375,15 @@ Frame Contents
     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
     |                         Stream ID = 0                         |
     +-----------+-+-+---------------+-------------------------------+
-    |Frame Type |0|M|     Flags     |
+    |Frame Type |0|0|     Flags     |
     +-----------+-+-+---------------+-------------------------------+
     |0|                       Time-To-Live                          |
     +---------------------------------------------------------------+
     |0|                     Number of Requests                      |
     +---------------------------------------------------------------+
-                                Metadata
 ```
 
 * __Frame Type__: (6 bits = max value 63) 0x02 
-* __Flags__: (10 bits)
```

Comment:
Sorry, I don't understand ... are you saying that a LEASE frame has a use case for including metadata? If so, what is it?